### DISCUSSION TITLE
OpenCV 강의자료

### DISCUSSION THREAD

**Discussion by kikiru328** [kikiru328/Bone_Detection#16] [Study]
## [파이썬을 활용한 데이터 분석과 이미지 처리] - 동빈나

- 강의자료
https://github.com/kikiru328/Python-Data-Analysis-and-Image-Processing-Tutorial

fork해서 사용하시면 될 것 같습니다

**Comment by kikiru328** [comment]
## [파이썬을 활용한 데이터 분석과 이미지 처리] - 동빈나
- youtube 주소
https://www.youtube.com/watch?v=V8Lpf3WCZ4g&list=PLRx0vPvlEmdBx9X5xSgcEk4CEbzEiws8C

**Comment by hye-jj** [comment]
## [ Python-OpenCV ] - 윤대희 

https://076923.github.io/posts/Python-opencv-2/

**Comment by yibnn** [comment]
## 파이썬으로 만드는 OpenCV 프로젝트(이세우 저)

https://github.com/BaekKyunShin/OpenCV_Project_Python


- 원본 깃허브(설명 살짝 불친절)
https://github.com/dltpdn/insightbook.opencv_project_python
-  블로그
https://bkshin.tistory.com/entry/OpenCV-13-%EC%9D%B4%EB%AF%B8%EC%A7%80-%EC%9D%B4%EB%8F%99Translation-%ED%99%95%EB%8C%80%EC%B6%95%EC%86%8CScaling-%ED%9A%8C%EC%A0%84Rotation

**Comment by hye-jj** [comment]
## [Python OpenCV] - 차준영
https://dsbook.tistory.com/category/%EC%B0%A8%EC%A4%80%EC%98%81/Python%20OpenCV

01. 이미지 reading and writing 
https://dsbook.tistory.com/143?category=802614
